content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -5,44 +5,159 @@ weight: 20
 
-lightness
-: This blend mode mixes lightness from the input and output images. Color (chroma and hue) is taken unaltered from the input image.
+# arithmetic modes
 
-chroma
-: This blend mode mixes chroma (saturation) from the input and output images. Lightness and hue are taken unaltered from the input image.
+addition
+: Add together the pixel values of the input and output images, lightening the output. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum".
 
-hue
-: This blend mode mixes hue (color tint) from the input and output images. Lightness and chroma are taken unaltered from the input image.
+subtract
+: Subtract the pixel value of the _output_ from the _input_. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum". Pixel values less than 0 are set to 0.
 
-color
-: This blend mode mixes color (chroma and hue) from the input and output images. Lightness is taken unaltered from the input image. 
+subtract reverse
+: _only available in the "RGB (Scene)" color space_
+: Subtract the pixel value of the _input_ from the _output_. In this case, the "blend fulcrum" parameter of the "RGB (scene)" color space is applied to the input image.
 
-: _Caution: When modules drastically modify hue (e.g. when generating complementary colors) this blend mode can result in strong color noise._
+multiply
+: Multiply the pixel values of the input and output together. As all pixel values are between 0 and 1.0, the resulting output will always be darker. When blending in the "RGB (scene)" color space, this value is further multiplied by a value proportional to the "blend fulcrum".
+
+multiply reverse
+: _only available in the "RGB (Scene)" color space_
+: This mode works in the same way as the "multiply" mode except that in the `final_output` equation given at the top of this page, the `input` parameter is replaced with `module_output` (the output of the module before blending is applied).
+
+divide
+: Divide the pixel values of the input by the output. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum".
+
+divide reverse
+: _only available in the "RGB (Scene)" color space_
+: Divide the pixel values of the _output_ by the _input_. In this case, the "blend fulcrum" parameter of the "RGB (scene)" color space is applied to the input image.
+
+screen
+: _not available in the "RGB (Scene)" color space_
+: Invert the input and output pixel values, multiply those values together and invert the result. This yields the opposite effect to "multiply" mode -- the resulting image is usually brighter, and sometimes “washed out” in appearance.
```

Comment:
replace "this is the opposite" by "this is approximately the opposite", because it is obviously not arithmetically equivalent and also assumes non-radiometrically-linear RGB.

Reply:
what is meant here by "invert"? I presume it can't mean taking the reciprocal, since 1/((1/a)\*(1/b)) = a\*b, and the description stayed this gives a different result to multiplication.

I can guess it might be something like 1 - (1-a)*(1-b), but the word "invert" is really ambiguous. this may be a good example justifying the inclusion of formulas in the descriptions.

Reply:
I copied a lot of this stuff from the GIMP documentation and other stuff I found.